modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -8240,6 +8178,42 @@ private void _testPutCustomObjectEntryUnlinkNestedCustomObjectEntries(
 		}
 	}
 
+	private void _testPutCustomObjectEntryWithDuplicateExternalReferenceCode(
```

Comment:
This method is almost equal to the `_testPatchCustomObjectEntryWithDuplicateExternalReferenceCode`:

Could you create just one? See this pattern: https://github.com/liferay/liferay-portal/blob/e9dee38587df6fe28b484f23cb5ae994e807aa47/modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java#L4738

![image](https://github.com/liferay-headless/liferay-portal/assets/32699538/40b19aba-674a-43b9-b4e7-7c7a7b02988f)